docs/post-mortems/0000-post-mortem-template.md

```diff
@@ -0,0 +1,109 @@
+## Summary
+<!--
+Describe the reason and the consequences of the event as short and concise as possible.
+
+Example:
+The backend of Oslo Nøkkelen were unavailable for 31 minutes due to an increase in traffic that happened after a
+proaktiv melding.
+-->
+
+## Timeline
+<!-- 
+Describe the relevant activites in a timeline format. Remember activities leading up to the event being triggered.
+
+2019-12-30
+    23:30 A proactive message got sendt out to all the citizens of Oslo
+2019-12-31
+    08:23 Oslonøkkelen had been downloaded 30.000 times, distributed evenly across iOS and Android, during the past 3 hours
+    09:09 DOWNTIME START - backend fails due to the increased load
+    09:14 Received Slack notifications due to an increase in 500 status codes in the backend
+    09:16 Initiated an investigation into the issue
+    09:18 EVENT START - Team creates the Slack channel #citykey-incident-backend to better cooperate on the issue
+    09:20 Manually scaling up the number of backend instances
+    09:24 Found the error. The backend fails when it tries to read an item in the database that doesn't exist. Only happens upon a users first login
+    09:28 Pull request with a fix created: http://github.com/oslokommune/something/something/pr/298
+    09:34 Pull request 298 merged and deployed to production
+    09:39 PROBLEM FIXED - Amount of 500 status code requests are decreasing
+    09:40 DOWNTIME END - Amount of 500 status codes are down to zero
+    10:10 EVENT END - Reaches criteria of 30 minutes with normal activity
+-->
+
+## Impact
+<!--
+Describe the consequences this had for the organization.
+
+Example:
+- Approximately 30.000 users downloaded and experienced an error using the app.
+- Users already logged in experienced some minor delay in the app
+-->
+
+## Cause(s)
+<!--
+Describe relevant factors that played a part in causing the event.
+
+Example:
+The error happened due to a combination of:
+- A spike of new users
+- A bug in the backend code for user creation
+-->
+
+## Solution
+<!--
+Describe how the problem was fixed.
+
+Example:
+- Manually scaled up number of backend instances letting regular traffic stay unaffected
+- Created a fix for the database schema
+-->
+
+## What went well
+<!--
+Describe what went well trying to handle the event.
+
+Example:
+- Alerting mechanisms worked brilliantly when errors started comming in
+- Deploying the database update was fast
+-->
+
+## What could we have done better
+<!--
+Describe what went wrong trying to handle the event.
+
+Example:
+- Had to manually roll back the database while testing fixes
+-->
+
+## Where were we lucky
+<!--
+Describe briefly the situations related to this event where we got lucky.
+
+Example:
+- Updating the database schema fixed the problem
+- Manual horizontal scale up ensured a small amount of users was affected
+-->
+
+## Mitigations
```

Comment:
I suggest removing mitigations. I feel it is the same as Action items, just not as specific

Reply:
I suggest keeping it. Here we are supposed to list all the things we can think of to mitigate the issue (brainstorming). Actions are what we decide to actually do (can be a subset, can be things that are not directly mitigations, etc.).

Reply:
Also, in incident reviews I've attended, the report is written beforehand by one or a few people, including a list of possible mitigations that they come up with. The action items are only decided upon by the full team during the actual review. (During the review you can also add to/change the mitigations list and other sections of course).

Reply:
So "mitigations" is a part of the review process, but "action items" is the value that "mitigations" results in?

If so I'd rather write a comment about it inside action items. I feel the template should only include information that is valueable looking back to. The process itself != the resulting document

Reply:
Action items are what we decide to do. Mitigations is what we could potentially do. Some of those might be relevant as actions later?

Reply:
Action items should not include things that would be "nice to have, but we won't be doing it now". Each line in action items should be a task in Trello.

Reply:
Yeah. Totally agree with action items should be tasks in Trello. I just have a hard time imagining having time enough to skim through old post-mortems looking for tasks that are nice to have :p

Reply:
Typically "Mitigations" will include a lot more than "Action items", and often the action items will be narrower/more specific. I think there is value in seeing the difference, without needing to look at the Git history or including a lot of comments in the "Action items" section that will just clutter it up.

Part of writing the report is writing a list of the potential mitigations you can come up with, so it's not like your'e saving any work by omitting the "Mitigations" section. Instead you are creating more work by needing to decide what should be left as comments, editing items to be more specific actions, etc.

Reply:
I think "Mitigations" can add value, even if nobody scand old post mortems for things to do. It can add useful info about what the post mortem author(s). Just by reading the points, I can get insight into the incident. It doesn't need to result in tasks now or in the future to provide that value.

The background for this opinion is my take on post mortem: I want to learn about important aspects of an incident, just to understand the nature of it. If the post mortem authors can't find any important to say for any point, then they're free to skip it.

Reply:
Leaving it